threat_intel/virustotal.py

```diff
@@ -9,174 +9,241 @@
 
 
 class VirusTotalApi(object):
-    BASE_DOMAIN = u'https://www.virustotal.com/vtapi/v2/'
+    BASE_DOMAIN = u'https://www.virustotal.com/api/v3/'
 
-    def __init__(self, api_key, resources_per_req=25, cache_file_name=None, update_cache=True, req_timeout=None):
+    def __init__(self, api_key, cache_file_name=None, update_cache=True, req_timeout=None):
         """Establishes basic HTTP params and loads a cache.
 
         Args:
             api_key: VirusTotal API key
-            resources_per_req: Maximum number of resources (hashes, URLs)
-                to be send in a single request
```

Comment:
Just confirm I understand correctly, we got rid of this cause the v3 API do not support querying multiple things at the same time, right? (we are going to burn through our API call allocations, sigh).

Reply:
yes, can confirm the batched requests are no longer supported:

`WARNING:root:Request to https://www.virustotal.com/api/v3/files/37e1446af71454bc73aa95a23ae50e9b88c4848260efaf8e9ec61522d1e23d0c,543864b4ba3e395e1eb0e73409bfdd23/download failed with status code 404. Response text: b'<html>\n <head>\n  <title>404 Not Found</title>\n </head>\n <body>\n  <h1>404 Not Found</h1>\n  The resource could not be found.<br /><br />\n\n\n\n </body>\n</html>'`